### ISSUE TITLE
Peer review deposit decision letter authors

### ISSUE THREAD

**Issue by gnott** [elifesciences/elife-crossref-feed#142]
In eLife article XML, the decision letter often includes an editor `<contrib>` tag. These can be parsed from the XML for inclusion in the peer review deposits to Crossref (however after todays' call, those editor values will probably be substituted from a data source external from the XML).

When searching existing XML, I do not find any decision letters where the authors of the decision letter are specified in `<contrib>` tags. I understand it is unlikely the reviewers for an article would be identified, but sometimes do the reviewers want to be identified as authors of the decision letter?

In terms of Crossref deposits, if we only include editor(s) in the decision letter peer review deposit, then when people use the Crossref metadata, that is the only contributor data associated with a decision letter.

Question being: Will there never be any authors associated with a decision letter peer review Crossref deposit? If there will ever be an author of a decision letter, where would that data come from?

@Melissa37 I thought you might be the first person to ask regarding this, and please feel free to share it with anyone else who might want to comment. Please feel free to ask questions or we can discuss if I didn't explain it very well to be understood!

**Comment by gnott** [comment]
I already have some additional points or questions.

Details from the page https://support.crossref.org/hc/en-us/articles/115005255706-Peer-Reviews, lead me to ask:

* Maybe `reviewer` as the contributor type might be also a possibility instead of `author` for a decision letter?
* Q. Would there be any value in declaring the decision letter author is `<anonymous/>`?

**Comment by Melissa37** [comment]
>In eLife article XML, the decision letter often includes an editor <contrib> tag. These can be parsed from the XML for inclusion in the peer review deposits to Crossref (however after todays' call, those editor values will probably be substituted from a data source external from the XML).

Yeah, I'd like to think this all through thoroughly from a current and future workflow point of view with more than just my brain :-)
 So:

- Currently, Exeter receive Author, Editor, Reviewer info in EJP XML and display it for proofing on the metadata page (not associated with the decision letter and author response in the UI)
- Occasionally something is spotted in Editor and Reviewer info, which requires an update (wrong person included, affiliation change). 
- More often something in the author list changes (spelling - less frequently adding/removing, changing order).
- Therefore the EJP metadata quickly loses its source of truth status after export.
- However, there is data in EJP that does not change and we do not receive in production - eg the date of the review

There are a few moving parts:

- We want to deliver Peer review metadata according to their schema asap

- We want to start sending Exeter the decision letter and author response in XML format for them to minimally process to display in Kriya

- We want to replace Kriya with something Texture like and for this to allow production access to edit all content for all purposes (it will be an XML interface)

- JATS4R has a working group on peer review materials

- EJP will be replaced by Libero Reviewer

These things are going to happen in stages, so we have limits in options that will widen/change as we get further along.

For Crossref delivery (1st moving part), perhaps the best option is to build the information based on what is in the final XML delivered from Exeter to the final bucket AND pull from EJP CSV where it cannot be found here (for example date of review).

Future:
eLife final XML contains all the information that will be sent to Crossref and that should be available for proofing in Texture
Decision letter and author response XML will contain all recommendations of JATS4R 

>When searching existing XML, I do not find any decision letters where the authors of the decision letter are specified in <contrib> tags. I understand it is unlikely the reviewers for an article would be identified, but sometimes do the reviewers want to be identified as authors of the decision letter?

We can safely assume that the authors of the paper are the authors of the decision letter and it should be all authors. Until JATS4R recommendation is finalised I suggest we don't start trying to add this to the eLife JATS XML, as it will also affect the UI decisions and Texture

>In terms of Crossref deposits, if we only include editor(s) in the decision letter peer review deposit, then when people use the Crossref metadata, that is the only contributor data associated with a decision letter.

@acollings are you OK with this? Or should we put in reviewer names that have agreed to release their names so they get credit as the letter is based on their efforts too?

>Will there never be any authors associated with a decision letter peer review Crossref deposit? 

The authors of this letter will be the editors +/- Reviewers. Not the authors of the original reasearch paper

>If there will ever be an author of a decision letter, where would that data come from?

The main article XML editor roles contribs. I'd like Andy to confirm.

>@Melissa37 I thought you might be the first person to ask regarding this, and please feel free to share it with anyone else who might want to comment. Please feel free to ask questions or we can discuss if I didn't explain it very well to be understood!

Do my answers help so far? It would be good to get wider input on it too :-)

**Comment by gnott** [comment]
Those are some valid points related to the simple question of who are the authors of a decision letter. 

I can see how potentially the authors of the research article being the authors of the decision letter may result in better meta data at Crossref. If reviewers are included as part of a decision letter, it is probably better for they to be set as reviewers, so as not to be confused as being the research article authors.

Whether the data is in the XML file at various stages of production, as you say, also impacts the decision letter parser library and assembling decision letter JATS.

Other tools what may be integrated into the procedure in the future, or alternate tools used by different publishers, adds additional complexity.

In the near term, I think I have enough actionable tasks to work on, which will still have some flexibility, and we can pivot if we want to.

Concerning the desire to deposit something "ASAP", we can probably re-deposit more elaborate peer review deposits later to replace any first version of deposits we start with. For the data we do not include right away, maybe we can create additional issues for those, and we can label them as `Deposit enhancements`, to be done at a later date?

**Comment by acollings** [comment]
@Melissa37 It might help to have a quick refresher about this, if that's okay. I will be in the office tomorrow and next week. E.g., usually we publish a summary of the reviews: in those cases, I assume we would send only that (and not the separate reviews) to CrossRef. In other cases we publish the separate reviews as part of the decision letter: in these instances, I assume we would send one package to CrossRef (rather than three separate ones, which would each have different authors). If any of this is mistaken or confusing, let's discuss.

But, if my assumptions are correct, my recommendation is to list the Reviewing Editor as the author of the decision letter package sent to CrossRef.

**Comment by Melissa37** [comment]
@acollings Good Idea. I've invited you, @JGilbert-eLife and @FAtherden-eLife to the @gnott / Melissa call next Tuesday at 4pm. Hope you can make it - I think it would be good for us all to discuss this. @joelsummerfield too

Thanks all!

**Comment by Melissa37** [comment]
From Crossref re overwriting old component DOIs:

>when you re-register a DOI, you retain the same DOI string but submit it with the new metadata. We’ll essentially wipe away the existing dataset record and replace it with a peer review record. There will be warnings in your submission log noting that you’ve changed content types, but beyond that it should be very simple.

**Comment by gnott** [comment]
I will close this issue, I think all the peer review deposits to Crossref get the author and editor names from data hub queries and not from the XML. FYI to @FAtherden-eLife, you will hopefully agree there's no more actions to be done for peer reviews at this time.
